server/datastore/mysql/scripts.go

```diff
@@ -719,6 +787,44 @@ func (ds *Datastore) UnlockHostViaScript(ctx context.Context, request *fleet.Hos
 	})
 }
 
+// WipeHostViaScript creates the script execution request and updates the
+// host_mdm_actions table in a single transaction.
+func (ds *Datastore) WipeHostViaScript(ctx context.Context, request *fleet.HostScriptRequestPayload) error {
```

Comment:
@ghernandez345 Another Datastore method you'll need to mock in the CLI Wipe test (for Linux hosts).